sfx_scripts/cmdseq.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+mkdir /tmp/input/ &&
```

Comment:
Use a name other than "mount" .

It seems like you need
/opt/collectd-source
/opt/collectd-build-source

Also there is no need to use /tmp. How about /opt/collectd-build